exporters/etw/include/opentelemetry/exporters/etw/etw_provider_exporter.h

```diff
@@ -0,0 +1,657 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#ifndef WIN32_LEAN_AND_MEAN
+#  define WIN32_LEAN_AND_MEAN
+#endif
+
+#ifdef _MSC_VER
+// evntprov.h(838) : warning C4459 : declaration of 'Version' hides global declaration
+#  pragma warning(disable : 4459)
+// needed for Unit Testing with krabs.hpp
+#  pragma warning(disable : 4018)
```

Comment:
Moving this `pragma` to Unit Testing header file before including this file?

Reply:
We can remove line 25, but we still need the pragma at 23. Best to handle this with pragma and pragma push-pop here rather than outside.